### ISSUE TITLE
Calling `grantOfflineAccess` requires user to sign in again?

### ISSUE THREAD

**Issue by JoeVanGundy** [google/google-api-javascript-client#664]
- I'm connecting to the Api to get some user account information before eventually sending the authorization code to my server.
- Once I have my authenticated user, I try to get an authorization code by calling `grantOfflineAccess`.
- This requires the user to sign in again.

Can someone confirm if that's expected behaviour?

**Comment by Lav-Hinsu** [comment]
Yes. ideally granting offline access allows an app//server to get access tokens infinitely. so you want user to know that they are granting offline access to their data. you cannot get the userObject and a authorization code in the same signIn session, they must signIn again.

**Comment by dvanderb** [comment]
This looks like a question about Google Login and Authentication. Please address this question to the google-signin tag on stackoverflow(https://stackoverflow.com/questions/tagged/google-signin)

If you believe this is in error, please reopen the issue.
